src/frontend/App.tsx

```diff
@@ -39,6 +41,15 @@ localDiscoveryController.start();
 initializeNodejs();
 SplashScreen.preventAutoHideAsync();
 
+export const queryClient = new QueryClient();
+
+function onInviteEvent() {
+  queryClient.invalidateQueries({queryKey: [INVITE_KEY]});
+}
+
+mapeoApi.invite.addListener('invite-received', onInviteEvent);
+mapeoApi.invite.addListener('invite-removed', onInviteEvent);
```

Comment:
didn't seem necessary to set this up in a hook + component but could be convinced otherwise